src/test/java/kitchenpos/bo/OrderBoTest.java

```diff
@@ -0,0 +1,219 @@
+package kitchenpos.bo;
+
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.OrderDao;
+import kitchenpos.dao.OrderLineItemDao;
+import kitchenpos.dao.OrderTableDao;
+import kitchenpos.model.Order;
+import kitchenpos.model.OrderLineItem;
+import kitchenpos.model.OrderStatus;
+import kitchenpos.model.OrderTable;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.BDDMockito.given;
+
+/**
+ * @author Geonguk Han
+ * @since 2020-02-15
+ */
+@ExtendWith(MockitoExtension.class)
+class OrderBoTest extends Fixtures {
+
+    @InjectMocks
+    private OrderBo orderBo;
+
+    @Mock
+    private MenuDao menuDao;
+
+    @Mock
+    private OrderDao orderDao;
+
+    @Mock
+    private OrderLineItemDao orderLineItemDao;
+
+    @Mock
+    private OrderTableDao orderTableDao;
+
+    @Test
+    @DisplayName("주문의 아이템이 없는 경우")
+    void create_empty_orderLineItem() {
+        final Order order = new Order();
+        order.setOrderLineItems(Collections.emptyList());
+
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> orderBo.create(order));
+    }
+
+    @Test
+    @DisplayName("주문의 아이템과 메뉴의 숫자가 일치하지 않는 경우")
+    void create_notEqual_menuItem_size() {
+        final Order order = new Order();
+        final OrderLineItem orderLineItem = new OrderLineItem();
+        orderLineItem.setMenuId(1l);
+
+        final OrderLineItem orderLineItem1 = new OrderLineItem();
+        orderLineItem1.setMenuId(2l);
+        order.setOrderLineItems(Arrays.asList(orderLineItem, orderLineItem1));
+
+        given(menuDao.countByIdIn(Arrays.asList(orderLineItem.getMenuId(),
+                orderLineItem1.getMenuId()))).willReturn(3l);
+
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> orderBo.create(order));
+    }
+
+    @Test
+    @DisplayName("주문 테이블이 존재하지 않는 경우")
+    void create_notExist_orderTable() {
+        final Order order = new Order();
+        final OrderLineItem orderLineItem = new OrderLineItem();
+        orderLineItem.setMenuId(1l);
+
+        final OrderLineItem orderLineItem1 = new OrderLineItem();
+        orderLineItem1.setMenuId(2l);
+        order.setOrderTableId(1l);
+        order.setOrderLineItems(Arrays.asList(orderLineItem, orderLineItem1));
+
+        given(menuDao.countByIdIn(Arrays.asList(orderLineItem.getMenuId(),
+                orderLineItem1.getMenuId()))).willReturn(2l);
+        given(orderTableDao.findById(order.getOrderTableId())).willThrow(IllegalArgumentException.class);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        given(orderTableDao.findById(order.getOrderTableId())).willReturn(Optional.empty());
```

Reply:
수정했습니다.